cEP-0001.md

```diff
@@ -4,38 +4,101 @@ Decision Making in coala
 |Metadata|                                                                    |
 |--------|--------------------------------------------------------------------|
 |cEP     |0001                                                                |
-|Version |1.1                                                                 |
+|Version |1.2                                                                 |
```

Comment:
Again, I don't feel like any of the old "behaviour" is actually changed but we can just make it 2.0 for the sake of making it obvious how big the change is.